skype/skype-ps/skype/Move-CsUser.md

```diff
@@ -323,22 +319,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-
-
 ### CommonParameters
+
 This cmdlet supports the common parameters: `-Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).`
 
 ## INPUTS
 
+### String
 
-###  
-String or Microsoft.Rtc.Management.ADConnect.Schema.ADUser object.
```

Comment:
Figured out these needed to be separate headings from the PlatyPS spec

Reply:
you rock